app/controllers/katello/api/v2/api_controller.rb

```diff
@@ -149,15 +149,15 @@ def scoped_search_query(query, group)
     end
 
     def scoped_search_total(query, group)
-      scoped_search_query(query, group).length
+      scoped_search_query(query, group).count
```

Comment:
Changing these from `length` to `count` improved response time by huge amounts, like ~ 2.5 seconds. I think it's because `count` does a SQL query where `length` is just Ruby calculating the length.